project_test/lib/pipeline_configurations.py

```diff
@@ -4,45 +4,63 @@
 
 class CodingPlan(object):
     def __init__ (self, raw_field, coded_field, coda_filename, cleaner=None, code_scheme=None, time_field=None,
-                    run_id_field=None, icr_filename=None, analysis_file_key=None, id_field=None):
+                    run_id_field=None, icr_filename=None, analysis_file_key=None, id_field=None,
+                    binary_code_scheme=None, binary_coded_field=None, binary_analysis_file_key=None):
         self.raw_field = raw_field
         self.coded_field = coded_field
-        self.coda.coda_filename = coda_filename
+        self.coda_filename = coda_filename
         self.icr_filename = icr_filename
         self.cleaner = cleaner
         self.code_scheme = code_scheme
         self.time_field = time_field
-        self.run_id_field = time_field
+        self.run_id_field = run_id_field
         self.analysis_file_key = analysis_file_key
+        self.binary_code_scheme = binary_code_scheme
+        self.binary_code_field = binary_code_field
+        self.binary_analysis_file_key = binary_analysis_file_key
+
 
         if id_field is None:
             id_field = "{}_id".format(self.raw_field)
-        self.run_id_field = id_field
+        self.id_field = id_field
 
 class PipelineConfiguration(object):
    
     DEV_MODE = True #Set to True when testing the pipeline - False removes data for AVF test contacts.
 
     TEST_SHOWS_CODING_PLAN = [
         CodingPlan(raw_field="internet_working_raw",
-                coded_field="internet_working_coded",
                 time_field="sent_on",
                 coda_filename="internet_working.json"
                 icr_filename="internet_working.csv",
                 run_id_field="internet_working_id",
-                analysis_file_key="internet_working_",
                 cleaner=None,
-                code_scheme=CodeSchemes.INTERNET),
+                binary_code_scheme=CodeSchemes.INTERNET,
```

Comment:
Do this you mean have a binary with no reasons as a radio question?

Reply:
Yes first two flows Water filter and Internet have no reason thats why I added a new demo question waste disposal that has a yes_no and reason.

Reply:
Ok. Advanced warning that the rest of the ReDSS pipeline will probably crash given a binary but no reasons. However, I think it should be straightforward to adapt to this use case.